src/js/actions/AppActions.js

```diff
@@ -35,8 +37,16 @@ const init = () => {
 
         const path = decodeURI(Player.getAudio().src).replace('file://', '');
```

Comment:
Yep, it was this line. It would fail on songs with ampersands in the path. I fixed it by changing it to `decodeURIComponent`. I think the reason that's needed is because of the corresponding `encodeURIComponent` [here](https://github.com/dkniffin/museeks/blob/master/src/js/utils/utils.js#L62)